website/docs/d/cloudant_database.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "ibm"
+page_title: "IBM : cloudant_database"
+description: |-
+  Get information about cloudant_database
+subcategory: "Cloudant Databases"
+---
+
+# ibm\_cloudant_database
+
+Provides a read-only data source for cloudant_database. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_cloudant_database" "cloudant_database" {
+	instance_crn  = var.instance_crn
```

Comment:
instead of var.XXX can you proivde an example which uses some kind of interpolation syntax reading from reosurce or datasource